chaoscenter/web/src/views/KubernetesChaosInfrastructureTable/KubernetesChaosInfrastructureTable.tsx

```diff
@@ -75,9 +75,10 @@ export function RenderChaosInfrastructureOptions({
         </Text>
         <Container>
           <CodeBlock
-            text={`kubectl delete chaosexperiment chaosengine chaosresult --all -n ${chaosInfrastructureNamespace}`}
+            text={`kubectl delete chaosexperiment,chaosengine,chaosresult --all -n litmus`}
```

Comment:
Can you revert `${chaosInfrastructureNamespace}`?

I know i have mentioned it issue for replaceing it with "litmus" but we wanted to make it dynamic and in that case the current one is true.